src/containers/shared/utils.js

```diff
@@ -139,9 +139,19 @@ export const localizeDate = (date, lang = 'en-US', options = {}) => {
   return new Intl.DateTimeFormat(lang, options).format(date);
 };
 
+/**
+ *
+ * @param {list} a The transactions in state.
+ * @param {list} b The new transactions from props.
+ * @returns {list} Concatenated list.
+ */
+// extract new items from top of array b using iterator
+// and merge it into the state array a.
```

Comment:
This should be above the params, e.g. in like 143

Reply:
Done!